<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgressDialog</class>
 <widget class="QDialog" name="ProgressDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>210</height>
   </rect>
  </property>
  <property name="sizePolicy">
   <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
    <horstretch>0</horstretch>
    <verstretch>0</verstretch>
   </sizepolicy>
  </property>
  <property name="minimumSize">
   <size>
    <width>400</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>400</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>RIVET: Computation in Progress</string>
  </property>
  <widget class="QPushButton" name="stopButton">
   <property name="geometry">
    <rect>
     <x>150</x>
     <y>170</y>
     <width>100</width>
     <height>27</height>
    </rect>
   </property>
   <property name="text">
    <string>Stop</string>
   </property>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>135</y>
     <width>381</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QLabel" name="step1description">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>15</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="font">
    <font>
     <weight>75</weight>
     <bold>true</bold>
    </font>
   </property>
   <property name="text">
    <string>Reading input file...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="step2description">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>37</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Building bifiltration...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="step3description">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>59</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Computing multigraded Betti numbers...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="step4description">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>81</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Building line arrangement...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="step5description">
   <property name="enabled">
    <bool>false</bool>
   </property>
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>103</y>
     <width>371</width>
     <height>21</height>
    </rect>
   </property>
   <property name="text">
    <string>Computing barcode templates...</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
